docs/svelte-testing-library/intro.md

```diff
@@ -58,13 +58,14 @@ describe('App', () => {
   })
 
   test('should change button text after click', async () => {
-    const { getByText } = render(App, { props: { name: 'world' } })
+    const { getByText, container } = render(App, { props: { name: 'world' } })
```

Comment:
@ematipico actually would you mind adding this in a separate section? for this basic example, I'd prefer to keep it as is, and then have 'containers' be listed below.

Reply:
```suggestion
    const { getByText } = render(App, { props: { name: 'world' } })
```